routing/RouteSolver.py

```diff
@@ -0,0 +1,129 @@
+##################################################################################
+##                                                                              ##
+##   Program: Basic Route Solver for Factory Roaming Robot                      ##
+##   Author: Collin Duddy                                                       ##
+##   Class: EECS_542 Senior Design University of Kansas                         ##
+##                                                                              ##
+##   Description: This program will take in user specified input of a graph,    ##
+##                or map of nodes. It will then calculate all routes possible   ##
+##                from the users start node to the users end-node. This data    ##
+##                will be piped to a Raspberry Pi that will then send the       ##
+##                data to an arduino for roaming the map.                       ##
+##################################################################################
+import Graph
+import argparse
+import yaml
+##################################################################################
+##                                                                              ##
+##                            Global Variables                                  ##
+##                                                                              ##
+##################################################################################
+graph = Graph.Graph()
+DIRECTIONS = ('N', 'S', 'E', 'W')
+##################################################################################
+##                                                                              ##
+##                            Functions/Methods                                 ##
+##                                                                              ##
+##################################################################################
+
+def load_config(config_file):
+    with open(config_file,"r") as ymlfile:
+        cfg = yaml.safe_load(ymlfile)
+    curr_node = ""
+    bad_chars = '(){}<>[]'
+    print(cfg)
+    for node in cfg['nodes']:
+        print(node)
+        graph.add_vertex(node['node']['name'])
+        for connected_nodes in node['node']['connected_nodes']:
+            print(connected_nodes)
+            graph.add_edge((node['node']['name'], connected_nodes['connected_node']))
+            #Add directional support here
+
+def bool_input(message):
+    '''From Adam Mertz'''
```

Comment:
Don't think we need this level of citation given that my comment was on the project itself.